src/NHSD.BuyingCatalogue.Solutions.Persistence/Repositories/SupplierRepository.cs

```diff
@@ -33,7 +33,15 @@ FROM Supplier
 
         // A full-text index is typically a better way of implementing this kind of search. For example, the LIKE search below will
         // always perform an index scan. Given the expected number of suppliers I doubt this will be a problem, however.
-        private const string GetSuppliersByNameSql = @"SELECT Id, [Name] FROM dbo.Supplier WHERE [Name] LIKE '%' + @name + '%' ORDER BY [Name];";
+        private const string GetSuppliersByNameSql = @"SELECT sup.Id, sup.[Name]
+     FROM dbo.Supplier AS sup
+		  INNER JOIN dbo.Solution AS sol
+		  ON sol.SupplierId = sup.Id
```

Comment:
You're right. We didn't discuss this particular area of formatting SQL queries in our last standards meeting, which reminds me that there were a couple of other small SQL formatting topics I forgot to cover last time.

I've always followed the format I've used above for join conditions. For me, it just seems to flow/read better with the left-hand column immediately following the table it belongs to. However, I can understand why you might prefer swapping the left and right-hand sides of the condition. (I'm deliberately avoiding using key types as it may not always be a particular type of key.) Anyway, I don't mind swapping them around.

We can always see what the consensus is with the rest of the team at the next standards meeting... I suspect you might be on to a winner with this one! 😃 

Like underscores in field names, I suspect we may never agree on aliases – they're good! 😃

Reply:
Agreed on the change & will raise in the coding standards meeting.